src/DssSpell.sol

```diff
@@ -21,32 +21,15 @@ import "dss-exec-lib/DssAction.sol";
 import "lib/dss-interfaces/src/dss/GemJoinAbstract.sol";
 import "lib/dss-interfaces/src/dss/IlkRegistryAbstract.sol";
 import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
-import "./CentrifugeCollateralValues.sol";
-
-interface Initializable {
-    function init(bytes32) external;
-}
-
-interface Hopeable {
-    function hope(address) external;
-}
-
-interface Kissable {
-    function kiss(address) external;
-}
-
-interface RwaLiquidationLike {
-    function ilks(bytes32) external returns (string memory,address,uint48,uint48);
-    function init(bytes32, uint256, string calldata, uint48) external;
-}
+import "lib/dss-interfaces/src/dss/VatAbstract.sol";
```

Comment:
We can remove `DSTokenAbstract`, and `GemJoinAbstract` so we don't have to include these in the verified output.

Reply:
Done.